pom.xml

```diff
@@ -9,6 +9,16 @@
     <version>1.0-SNAPSHOT</version>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>3.0.0-M3</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
```

Comment:
Why do you need two surefire-plugin in pom?

Reply:
fixed